Microsoft.ML.sln

```diff
@@ -427,6 +431,14 @@ Global
 		{8B38BF24-35F4-4787-A9C5-22D35987106E}.Release|Any CPU.Build.0 = Release|Any CPU
 		{8B38BF24-35F4-4787-A9C5-22D35987106E}.Release-Intrinsics|Any CPU.ActiveCfg = Release|Any CPU
 		{8B38BF24-35F4-4787-A9C5-22D35987106E}.Release-Intrinsics|Any CPU.Build.0 = Release|Any CPU
+		{5A79C7F0-3D99-4123-B0DA-7C9FFCD13132}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{5A79C7F0-3D99-4123-B0DA-7C9FFCD13132}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{5A79C7F0-3D99-4123-B0DA-7C9FFCD13132}.Debug-Intrinsics|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I've merged my PR: [https://github.com/dotnet/machinelearning/pull/1032](https://github.com/dotnet/machinelearning/pull/1032).  These will need to be updated to have `-Intrinsics` on the right side too. See the PR for what needs to be done.